core/base/configmaps.yaml

```
@@ -36,3 +36,11 @@ kind: ConfigMap
 apiVersion: v1
 metadata:
   name: prometheus
+---
+# Change webhook and split into test, stage, prod once available
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: qeb-hwt-workflow
+data:
+  webhook-callback-url: "http://qeb-hwt-webhook-receiver-aicoe-prod-bots.apps.ocp.prod.psi.redhat.com"
```

```suggestion
  webhook-callback-url: " "
```

maybe this isnt the best place to add this, i feel
as this is subject to the cluster(in this case ocp) having it defined in the base, would cause it to be present in all deployments
and have another configmap in stage,or test deployment which assigns the value.

---

Totally make sense, thanks Harshad! So I copy it for test and stage separately for now?

---

yes , that would be good for us.